src/Tasks/Microsoft.NET.Build.Tasks/PrepareForReadyToRunCompilation.cs

```diff
@@ -139,7 +139,7 @@ void ProcessInputFileList(ITaskItem[] inputFiles, List<ITaskItem> imageCompilati
                 if (!InputFileEligibleForCompilation(file))
                     continue;
 
-                var outputR2RImageRelativePath = file.GetMetadata(MetadataKeys.RelativePath);
+                var outputR2RImageRelativePath = file.GetMetadata("RelativePath");
```

Comment:
Should we not have kept `MetadataKeys.RelativePath`?  When I saw the change to `MetadataKeys` earlier, I thought `PathInPackage` was going to replace all uses of this metadata key.

Reply:
Originally I had replaced all occurrences, but #2997 was merged while I was working on this PR, adding a use of RelativePath.  I'm not entirely sure that couldn't be changed to PathInPackage, but I left it as RelativePath and the crossgen tests passed, so I think it was probably right.

It is reasonable to add it back to MetadataKeys now though.